debian/changelog

```diff
@@ -1,3 +1,39 @@
+mir (2.4.0) UNRELEASED; urgency=medium
+
+  [ William Wold ]
+  * New upstream release 2.4.0
+
+    - ABI summary:
+      . mirclient ABI unchanged at 10
+      . miral ABI unchanged at 4
+      . mirserver ABI bumped to 55
+      . mircommon ABI unchanged at 7
+      . mirplatform ABI bumped to 22
+      . mirprotobuf ABI unchanged at 3
+      . mirplatformgraphics ABI unchanged at 18
+      . mirinputplatform ABI unchanged at 8
+      . mircore ABI unchanged at 1
+      . mircookie ABI unchanged at 2
+      . mirwayland ABI unchanged at 2
+    - Enhancements:
+      . [gbm-kms] Also respect --driver-quirks in the RenderingPlatform
```

Comment:
This is a fix/follow-up for the initial `--driver-quirks` work, which we haven't mentioned.

This should probably be something like:
[gbm-kms] Add --driver-quirks option to avoid probing devices where probing crashes